gpkit/nomials.py

```diff
@@ -454,9 +489,14 @@ def __pow__(self, other):
 
 
 class Constraint(Posynomial):
-    '''
-    TODO: Add docstring
-    '''
+    """A constraint of the general form monomial > posynomial
+    Stored internally (exps, cs) as a single Posynomial (1 >= self)
+    Usually initialized via operator overloading, e.g. cc = y**2 >= 1 + x
+    Additionally stores input format (lhs vs rhs) in self.right and self.left
+    Form is self.left >= self.right.
```

Comment:
The last line, `Form is self.left >= self.right.` is technically not correct due to the current logic for choosing left and right, correct?

@bqpd can you think of a better way to describe this? Any interest in removing this logic, simplifying things so that `left` is always what the user put on the greater than side?